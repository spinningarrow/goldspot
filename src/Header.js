import React from 'react'

const Header = () =>
	<header>
		<h1><span>Gold</span>spot</h1>
	</header>

export default Header
