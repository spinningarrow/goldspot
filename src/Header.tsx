import React from 'react'

const Header = ({ secretAction }) => (
	<header
		onClick={() => {
			secretAction(
				(prompt('Which features do you want to enable?') || '').split(',')
			)
		}}
	>
		<h1>
			<span>Gold</span>spot
		</h1>
	</header>
)

export default Header
