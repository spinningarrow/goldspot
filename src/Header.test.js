import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'

import Header from './Header'

const renderer = new ShallowRenderer()

test('it renders a heading with the app name', () => {
	const tree = renderer.render(<Header />)
	expect(tree).toMatchSnapshot()
})
