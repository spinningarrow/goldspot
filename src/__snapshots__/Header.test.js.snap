// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders a heading with the app name 1`] = `
<header
  onClick={[Function]}
>
  <h1>
    <span>
      Gold
    </span>
    spot
  </h1>
</header>
`;
